File: ITS320_Basic_Programming/tjs_code/ITS320-portfolio/Car.py
#Author: Thomas Keefe
#Email: tjk819@gmail.com
#Date: 12/03/2023

class Car:
    #Create an automobile class that will be used by a dealership as a vehicle inventory program. 
    #The following attributes should be present in your automobile class
    #
    #private string make
    #private string model
    #private string color
    #private int year
    #private int mileage

    #Your program should have appropriate methods such as:

    #constructor

    def __init__(self, make, model, color, year, mileage) -> None:
        self.make = make
        self.model = model
        self.color = color
        self.year = int(year)
        self.mileage = int(mileage)    
        
    def __str__(self) -> str:
        return f"{self.year} {self.make} {self.model}, {self.color}, {self.mileage} miles"
    
    def get_make(self):
        return self.make

    def get_model(self):
        return self.model
    
    def get_color(self):
        return self.color
    
    def get_year(self):
        return self.year
    
    def get_mileage(self):
        return self.mileage
    
    def set_make(self, make):
        self.make = make

    def set_model(self, model):
        self.model = model

    def set_color(self, color):
        self.color = color

    def set_year(self, year):
        self.year = year

    def set_mileage(self, mileage):
        self.mileage = mileage

File: CSC372_Programming_2/CSC372_Portfolio_Project/Car.java

public class Car {

	private String make;
	private String model;
	private double mpg;
	
	public String getMake() {
		return make;
	}

	public void setMake(String make) {
		this.make = make;
	}

	public String getModel() {
		return model;
	}

	public void setModel(String model) {
		this.model = model;
	}

	public double getMpg() {
		return mpg;
	}

	public void setMpg(double mpg) {
		this.mpg = mpg;
	}

	public Car(String make, String model, double mpg) {
		this.make = make;
		this.model = model;
		this.mpg = mpg;
	}
	
	public String toString() {
		return String.format("%s %s, %.2f mpg\n", make, model, mpg);
	}

}
